display/ssd1306.h

```diff
@@ -203,6 +206,8 @@ class SSD1306 : public I2CDevice, Looper, StateMachine, SaberBase, private Audio
 
       case SCREEN_PLI:
         memset(frame_buffer_, 0, sizeof(frame_buffer_));
+      // Need non-SDcard content at first so Mass Storage can work.
+          if (!just_booted_) ShowFile(&IMG_idle, font_config.ProffieOSOnImageDuration);
```

Comment:
It seems weird that we don't do an } else { or something here to avoid doing a showfile AND draw a battery bar.

Reply:
It works perfectly like this in my testing. Unless you mean in the background it's ALSO drawing battery bar?

Reply:
Fixed.